src/Microsoft.VisualStudio.Editors/PropPages/Strings.resx

```diff
@@ -131,4 +131,15 @@
     <value>http://go.microsoft.com/fwlink/?LinkID=287120</value>
     <comment>{Locked}</comment>
   </data>
+  <data name="LanguageVersionAutomaticallySelected" xml:space="preserve">
+    <value>Automatically selected based on framework version</value>
+    <comment>Refers to the C# langauge version being automatically selected based on the target framework</comment>
+  </data>
+  <data name="CantSelectLanguageVersionLinkLabel" xml:space="preserve">
+    <value>Why can't I select a different C# version?</value>
+  </data>
+  <data name="CantSelectLanguageVersionFWLink" xml:space="preserve">
+    <value>http://blah</value>
```

Comment:
Please note that I won't merge this in until @cartermp has created the actual link for this.

Reply:
aka.ms?

Reply:
Yeah, it's going to be an aka.ms link, it just hasn't been determined yet.

Reply:
`aka.ms/csharp-versions` is now live.

The URL currently points to the `LangVersion` section of our docs, but I'll update that once the actual content is written.

Reply:
Fun story, this is how I got that URL:

* Edge --> website didn't load
* Chrome --> website didn't load
* Edge --> turn off all extensions --> website didn't load
* Chrome --> turn off all extensions --> website didn't load
* Firefox --> worked great, including with all of my extensions that block everything

Reply:
Updated.